app/templates/admin_template.html

```diff
@@ -147,35 +147,73 @@
     {% set meta_suffix = 'Built by the <a href="https://www.gov.uk/government/organisations/government-digital-service" class="govuk-footer__link">Government Digital Service</a>' %}
   {% endif %}
 
-  {{ notify_footer({
+  {% if signed_out %}
```

Comment:
You can use the `current_user` global so that this logic applies to any page:
```suggestion
  {% if not current_user.is_authenticated %}
```
https://github.com/alphagov/notifications-admin/blob/d73c3a68f9ec3468842b4b18483d82ffa7d211ac/app/templates/admin_template.html#L44

Reply:
I made the change (thanks for that, using existing code instead of new == awesome) but with the condition reversed so you don't have to deal with a `not`.